### PR TITLE
added channel_id to PushMessage for new Android Notification Channels

### PR DESCRIPTION
**Pull request by earle**

### CODE REVIEW

**Comment by anp** [comment]
Hi! Thanks for sending this over! I've added @ccheever as a reviewer on this, he should be able to take a look soon.

File: exponent_server_sdk/__init__.py [hunk 1]
```diff
@@ -118,6 +121,9 @@ def get_payload(self):
             payload['priority'] = self.priority
         if self.badge is not None:
             payload['badge'] = self.badge
+        if self.channel_id is not None:
+            payload['channel_id'] = self.channel_id
```
Comment (ide) [inline exponent_server_sdk/__init__.py:125]: The field in the JSON-encoded payload sent to the server should be channelId instead of channel_id

**Review by ide — CHANGES_REQUESTED** [review]

**Comment by raphaelrk** [comment]
Fixed the channel_id vs channelId issue in #14

**Comment by wli** [comment]
closing in favor of #14. Thanks for the contribution regardless!

**Pull request closed without merging**